server/routes/creatingLicences/types/time.ts

```diff
@@ -25,10 +25,7 @@ class SimpleTime extends Stringable {
   ampm: AmPm
 
   stringify(): string {
-    if (this.hour && this.minute && this.ampm) {
```

Comment:
No longer needed because the licence service checks for an empty object before persisting to database, so no need to convert an empty object to an empty string